src/views/Dashboard.vue

```diff
@@ -26,11 +26,11 @@
         </v-card>
       </v-flex>
       <v-flex xs12 md3>
-        <v-card class="VCard ma-1" text justify-center dark color="primary">
+        <v-card class="VCard ma-1" text justify-center dark color="primary" @click="RedirectPublic()">
           <v-card-title class="display-1">See Public Ideas</v-card-title>
           <v-card-actions>
             <v-spacer></v-spacer>
-            <v-btn @click="RedirectPublic" depressed text fab>
+            <v-btn @click="RedirectPublic()" depressed text fab>
```

Comment:
Please consider remove `@click="RedirectPublic()"` cuz we are already calling the function on the parent `<v-card>`